File: backend/Event.py
class Event:
    def __init__(
            self, 
            id,
            group_id,
            event_type_id, 
            title, 
            all_day,
            start,
            end,
            rrule,
            url,
            editable,
            extended_props
    ):
        self.id = id
        self.group_id = group_id
        self.event_type_id = event_type_id
        self.title = title
        self.all_day = all_day
        self.start = start
        self.end = end
        self.rrule = rrule
        self.url = url
        self.editable = editable
        self.extended_props = extended_props

File: src/interfaces/Event.ts
export interface EventInterface{
    id: number,
    group_id: number,
    event_type_id: number,
    title: string,
    all_day: boolean,
    start: string,
    end: string,
    rrule: string,
    url: string,
    editable: boolean,
    extended_props: string
}

export interface EventTypeInterface{
    id: number,
    name: string,
    color: string,
    backgroundColor: string
}

export interface EventGroupInterface{
    id: number,
    name: string
}